app/models/concerns/dynamic_field_structure/string_key.rb

```diff
@@ -17,7 +17,12 @@ module StringKey
       # Validate that the string_key is unique between all siblings.
       def unique_string_key
```

Comment:
@barmintor If we're relying on a unique constraint to validate this rare (but important to protect against) error, and handling it for error display, I'm now wondering if we even need to keep `validate :unique_string_key` or the `unique_string_key` method at all.